novuRajan/gannt-chart
# AC2-gantt-133-Implement-Concurrent-Execution-in-npm-Run-Dev

The existing project setup lacks concurrent execution in the `npm run dev` command, hindering development efficiency.

**Solution:**
Implemented concurrent execution for TypeScript compilation in watch mode (`npm run start-ts`) and starting an HTTP server (`npm run start-server`) within the `npm run dev` command.

## Changes Made

- Modified the package.json file to run TypeScript compilation and start the HTTP server concurrently.

## Testing Instructions

1. Run `npm install` to ensure all dependencies are installed.
2. Execute `npm run dev`.
3. Verify that both TypeScript compilation in watch mode and the HTTP server start concurrently.
4. Ensure changes in TypeScript files trigger recompilation.
5. Confirm that the HTTP server is accessible at the specified port.
